ci.sh

```diff
@@ -115,7 +115,7 @@ else
         # when installing, and then running 'certmgr.msc' and exporting the
         # certificate. See:
         #    http://www.migee.com/2010/09/24/solution-for-unattendedsilent-installs-and-would-you-like-to-install-this-device-software/
-        certutil -addstore "TrustedPublisher" .github/workflows/astrill-codesigning-cert.cer
+        certutil -addstore "TrustedPublisher" trio/tests/astrill-codesigning-cert.cer
```

Comment:
Hm, I'm not too sure of this rename. It isn't getting out into the sdist right?

Reply:
I suppose we could prune it from MANIFEST.in, but at least [one person thinks it aught to go in there](https://github.com/python-trio/trio/pull/2541#issuecomment-1400712472)! Do you have a specific suggestion for a better home for it if we are not going to include it in the sdist?

Reply:
I don't have much experience with these so maybe it would be best to include it in the sdist...

But that feels weird. I would rather a certificate like this be in a top-level `assets` directory (maybe rename the top-level `logo` directory?).

Reply:
Just to be clear: in my mind this certificate is just like a pytest plugin -- it may be required to run (some) tests but it certainly shouldn't weigh everyone who installs the package down.

Reply:
I just built a wheel and sdist on my machine and they do not pick up the cert, so I think this question can be deferred to #2541

Reply:
Alright. I now realize I could have checked CI logs but... whatever.